renku/ui/cli/utils/terminal.py

```diff
@@ -65,26 +66,26 @@ def print_markdown(text: str, err: bool = False):
     Console(stderr=err).print(Markdown(text))
 
 
-def print_key_value(key, value, print_empty: bool = True, err: bool = False):
+def print_key_value(key, value, print_empty: bool = True, err: bool = False, indent: str = ""):
```

Comment:
Maybe having `indent` be an `int` and having `click.echo("  " * indent + style_key(key) + style_value(value), err=err)` is cleaner